diff -crN ./nginx-1.11.9.orig/src/http/ngx_http_core_module.h ./nginx-1.11.9/src/http/ngx_http_core_module.h
*** ./nginx-1.11.9.orig/src/http/ngx_http_core_module.h 2017-02-07 10:51:41.499571497 +0000
--- ./nginx-1.11.9/src/http/ngx_http_core_module.h      2017-02-07 10:53:42.818532753 +0000
***************
*** 572,582 ****
      }

  #define ngx_http_clear_etag(r)                                                \
-                                                                               \
-     if (r->headers_out.etag) {                                                \
-         r->headers_out.etag->hash = 0;                                        \
-         r->headers_out.etag = NULL;                                           \
-     }
-

  #endif /* _NGX_HTTP_CORE_H_INCLUDED_ */
--- 572,576 ----
